lib/express-rate-limit.js

```diff
@@ -9,6 +9,7 @@ function RateLimit(options) {
       message: "Too many requests, please try again later.",
       statusCode: 429, // 429 status = Too Many Requests (RFC 6585)
       headers: true, //Send custom rate limit header with limit and remaining
+      draft_polli_ratelimit_headers: false, //Support for the new RateLimit standardization headers
```

Comment:
Does not seem a great property name to me, maybe let's use the draft number instead of `polli`?

Reply:
To be honest, I suggested `draft_chickens` :)

Reply:
Yea, I thought about this a bit. I don't really like it for a permament name, but it isn't meant to be. Once the draft becomes an actual standard, I'm going to rename it to just `headers` and drop or rename the existing `headers` param. 

I think this name helps to both tie it to the draft and reinforce the idea that this is temporary and will change in the future.

The official id looks to be [draft-polli-ratelimit-headers-01](https://tools.ietf.org/id/draft-polli-ratelimit-headers-01.html) but I switched to underscores for ergonomics and dropped the number because I don't anticipate any significant changes. If there are some, we can add a `draft_polli_ratelimit_headers_02` param or whatever.